src/kustomization.yaml

```diff
@@ -1,12 +0,0 @@
----
-apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I would keep the file, so if we add future pipelines the install command stays the same